Components: Add README for layout related components (#24804)
Automattic/jetpack: projects/js-packages/components/components/layout/README.md

# Layout

> This is our components and utilities to compose interfaces, it's based in our current specs for Layout composition and CSS Grid Layout.

Reviewer: ```suggestion
These are our components and utilities to compose interfaces, it's based on our current specs for Layout composition and CSS Grid Layout.
```
